docs.html

```diff
@@ -0,0 +1,552 @@
+            </span>
+
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+                </code>
+            </pre>
+        </div>
+
+        <!--Text-->
+        <h2 id="text">Text</h2>
+        <h4>Example</h4>
+        <div class="div-example">
+            <h1 class="heading">Heading 1</h1>
+            <h2 class="heading">Heading 2</h2>
+            <h3 class="heading">Heading 3</h3>
+            <h4 class="heading">Heading 4</h4>
+            <h5 class="heading">Heading 5</h5>
+            <h6 class="heading">Heading 6</h6>
+
+
+            <h3>Small Text</h3>
+            <p class="small-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia, maxime,
+                praesentium
+                quas
+                esse, fugit vel
+                velit aperiam explicabo ad ipsa adipisci? Voluptatum dolorem quod cum omnis, voluptates autem
+                numquam
+                consequatur?
+            </p>
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+        &lth1 class="heading"&gtHeading 1&lt/h1&gt
+        &lth2 class="heading"&gtHeading 2&lt/h2&gt
+        &lth3 class="heading"&gtHeading 3&lt/h3&gt
+        &lth4 class="heading"&gtHeading 4&lt/h4&gt
+        &lth5 class="heading"&gtHeading 5&lt/h5&gt
+        &lth6 class="heading"&gtHeading 6&lt/h6&gt
+
+
+        &lth3&gtSmall Text&lt/h3&gt
+        &ltp class="small-text"&gtLorem ipsum dolor sit amet consectetur adipisicing
+        praesentium
+        quas
+        esse, fugit vel
+        velit aperiam explicabo ad ipsa adipisci? Voluptatum dolorem quod cum 
+        numquam
+        consequatur?
+        &lt/p&gt
+                </code>
+            </pre>
+        </div>
+
+
+
+
+    </div>
+    <footer class="footer">
+        <p>This Component Library is Made by <a href="https://twitter.com/shahazad__">Shahazad</a></p>
+    </footer>
+</body>
+
+</html>
```

Comment:
I'd suggest to move the documentation of individual components to their separate HTML files, so that it's easier to maintain when you add more components